index.html

```diff
@@ -30,7 +30,7 @@
             <img src="img/gamer 1.png" alt="" />
```

Comment:
Why did you use two google fonts imports?

Reply:
Done